packages/bff/src/utilities/config.ts

```diff
@@ -56,11 +61,35 @@ export type PurposeProcessServerConfig = z.infer<
   typeof PurposeProcessServerConfig
 >;
 
+export const AuthorizationProcessServerConfig = z
+  .object({
+    TENANT_ALLOWED_ORIGINS: z.string(),
+  })
+  .transform((c) => ({
+    tenantAllowedOrigins: c.TENANT_ALLOWED_ORIGINS.split(","),
+  }));
+
 const BffProcessConfig = CommonHTTPServiceConfig.and(TenantProcessServerConfig)
   .and(AgreementProcessServerConfig)
   .and(CatalogProcessServerConfig)
   .and(AttributeRegistryProcessServerConfig)
-  .and(PurposeProcessServerConfig);
+  .and(PurposeProcessServerConfig)
+  .and(AuthorizationProcessServerConfig)
+  .and(TokenGenerationConfig)
+  .and(FileManagerConfig)
+  .and(
+    z
+      .object({
+        ALLOW_LIST_CONTAINER: z.string(),
+        ALLOW_LIST_PATH: z.string(),
+        ALLOW_LIST_FILE_NAME: z.string(),
+      })
+      .transform((c) => ({
+        allowListContainer: c.ALLOW_LIST_CONTAINER,
+        allowListPath: c.ALLOW_LIST_PATH,
+        allowListFileName: c.ALLOW_LIST_FILE_NAME,
+      }))
```

Comment:
I don't know if this configurations could be useful for future services, anyway I suggest to create ad hoc config type like others

Reply:
This is specific to BFF and it is defined in the `BffProcessConfig` that is this file